[#74339] 17.4 Search improvements  https://community.openproject.org/… (#22996)
opf/openproject: docs/user-guide/search/README.md

>
> It is not possible to find content from archived projects. You will have to un-archive them first on the [global projects overview page](../projects/project-lists/).

## Partial-text search

> The OpenProject search bar is now also configured to support partial-text search. This means you no longer need to enter  the full word to find relevant results. Simply type part of a word, and the system will display matching items.

Reviewer: @ehassan01 I would not use "now", "no longer", etc., since we do not know when a user might refer to documentation. It makes sense in the release notes to describe features specific to a certain release, but in the user guide I'd advise against it.

For example: Entering "**de**" can return results such as **Design**, entering **"roto"** can also return results such as **Prototyping**.

> 

Reviewer: Could you add a screenshot here to illustrate this?

Author: > Could you add a screenshot here to illustrate this?

Hi @MayaBerd I have attached some image reference for partial text, where, keyword: **roto** matches prototyping. For multi-spaced text: **de roto** matches design prototyping. The example of marketing plan can be discarded.
<img width="3060" height="4080" alt="IMG_20260504_114421" src="https://github.com/user-attachments/assets/aabad02a-4e6b-4ba8-9530-076f7bcf5e17" />
<img width="4080" height="3060" alt="IMG_20260504_113935" src="https://github.com/user-attachments/assets/2012e570-ab06-404c-8da4-74ae6942136e" />
<img width="3060" height="4080" alt="IMG_20260504_114434" src="https://github.com/user-attachments/assets/fdc6d9bc-44a9-4bf9-a4b0-00169b96e4d3" />

Reviewer: Thanks @ehassan01, I rather meant if you could add screenshots to the PR, to illustrate the function in the same style as on the rest of the page.

This feature is especially helpful if you do not remember the exact name of a work package or are unsure of the first few letters.

## Multi-text search

You can also search using multiple space-separated texts to narrow down results.

For example: Entering "**ma pl"** will match **Marketing Plan**.

> 

Reviewer: Could you add a screenshot here to illustrate this?

>
> It is not possible to find content from archived projects. You will have to un-archive them first on the [global projects overview page](../projects/project-lists/).

## Partial-text search

The OpenProject search bar is also configured to support partial-text search. Simply type part of a word, and the system will display matching items.

For example: Entering **"roto"** will return results such as **prototyping**, **prototype**

> This feature is especially helpful if you do not remember the exact name of a work package or are unsure of the first few letters.

Reviewer: ```suggestion
This feature is especially helpful if you do not remember the exact name of a work package or are unsure of the exact spelling.
```

![Partial text search for work package search in OpenProject](openproject_userguide_search_partial_text.png)

> ## Multi-text search

Reviewer: ```suggestion
## Multi-term text search
```

> You can also search using multiple space-separated texts to narrow down results.

Reviewer: ```suggestion
You can also search using multiple space-separated terms to narrow down results.
```

> For example: Entering "**de roto"** will match **Design prototyping**.

Reviewer: ```suggestion
For example: Entering **"de roto"** will match **Design prototyping**.
```

> For example: Entering "**de roto"** will match **Design prototyping.

Reviewer: ```suggestion
For example: Entering "**de roto"** will match **Design prototyping**.
```

> When you type more than one text, the search will only show results where **all entered items** appear somewhere in the project or workspace name. This helps you find items faster, even if you only remember parts of the name.

Reviewer: ```suggestion
When you type more than one text, the search will only show results where **all entered terms** appear somewhere in the project or workspace name. This helps you find items faster, even if you only remember parts of the name.
```

> ![Multi space text for work package search in OpenProject](openproject_userguide_search_multispace_text.png)

Reviewer: ```suggestion
![Multi term text search for work package search in OpenProject](openproject_userguide_search_multispace_text.png)
```
